src/components/forms/forms.js

```diff
@@ -288,14 +292,20 @@ const formInputSlider = (config) => {
 */
 export const FormField = (config) => {
   const {
-    id, title, hideTitle, description,
+    id, title, hideTitle, description, type,
     defaultValue, style, validators
   } = config;
 
   const [error, setError] = useState();
-  const [formValue, setFormValue] = useState(defaultValue || '');
+  const [formValue, setFormValue] = useState(type?.defaultValue || '');
   const required = validators ? validators.findIndex(v => v === FormValidators.REQUIRED) !== -1 : false;
```

Comment:
`validators.includes(FormValidators.REQUIRED)`?

Reply:
same comment other usages of this

Reply:
updated!